rps_game.js

```diff
@@ -28,9 +28,10 @@ function playGame() {
   let userMove = userGesture();
   let computerMove = compGesture();
 
-  const result = document.getElementById('result');
-
+  let robotFace = document.getElementById('robot_face').src;
+  let humanFace = document.getElementById('human_face').src;
```

Comment:
you don't need these definitions, you're not using them :slightly_smiling_face: 

or if you want to create them to access them later, you have to remove `.src` at the end, or you would be storing the value of src instead of storing the full HTML element :+1:

Reply:
forgot to delete those, sorry